src/xdocs/config_javadoc.xml

```diff
@@ -814,6 +736,47 @@ public int checkReturnTag(final int aTagIndex,
   &lt;property name="excludeScope" value="protected"/&gt;
 &lt;/module&gt;
         </source>
+
+        <p>
+          To configure the check to validate <code>throws</code> tags, you can use following config.
+          ATTENTION: checkstyle do not have information about hierarchy of exception types so usage
+          of base classes will be considered as separate exception type. As workaround you need to
+          specify both types base and exact type in javadoc.
+        </p>
+        <source>
+&lt;module name="JavadocMethod"&gt;
+  &lt;property name="validateThrows" value="true"/&gt;
+&lt;/module&gt;
+        </source>
+        <source>
+/**
+ * Actually throwed is child class of class that is declared in throws.
+ * It is limitation of checkstyle (as we do not know type hierarchy).
+ * Code and javadoc is valid BUT checkstyle can not distinguish it.
```

Comment:
`Code and javadoc is valid` => `Javadoc is valid not declaring {@link FileNotFoundException}`.

I don't see why original was specifying code when it is all about the javadoc.

Reply:
done

Reply:
I removed `@link` as it require import (jdk11 failure and TC faliure)